packages/devtools_app/lib/src/screens/network/network_screen.dart

```diff
@@ -248,6 +252,111 @@ class _NetworkProfilerControlsState extends State<_NetworkProfilerControls>
             widget.controller.clear();
           },
         ),
+        const SizedBox(width: denseSpacing),
+        ExportNetButton(
+          minScreenWidthForTextBeforeScaling:
+              _NetworkProfilerControls._includeTextWidth,
+          onPressed: () {
```

Comment:
put this entire onPressed function into a method on the controller _exportAsHarFile.